src/proxy_cli.rs

```diff
@@ -0,0 +1,44 @@
+use anyhow::Result;
+use std::ffi::OsString;
+use std::os::unix::prelude::CommandExt;
+use std::process::{Command, ExitCode, Stdio};
+use std::{env, io};
+
+use crate::path::settings_file;
+use crate::settings::SettingsFile;
+use crate::toolchain::Toolchain;
+
+fn is_supported_plugin(plugin: &str) -> bool {
+    ["fmt", "lsp", "explore"].contains(&plugin)
```

Comment:
We should likely do something similar to what we do with components for plugins and use consts.

Reply:
Replaced with `SUPPORTED_PLUGINS` in [ed2fcc5](https://github.com/FuelLabs/fuelup/pull/73/commits/ed2fcc511f8a16db69339dd15cfaac675d04548f)